examples/catalog-module/index.js

```diff
@@ -1,13 +1,20 @@
-import {render} from '../../src/index';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Catalog} from '../../src/index';
 
-render({
-  title: 'My Components',
-  pages: [
-    {
-      path: '/',
-      title: 'Foo',
-      imports: {Foo: require('./components/Foo/Foo')},
-      component: require('./components/Foo/Foo.docs.md')
-    }
-  ]
-}, document.getElementById('app'));
+ReactDOM.render(
```

Comment:
This exposes React Router to the outside world, however, but I think it should be an implementation detail that stays within Catalog. I love the symmetry here:

```
ReactDOM.render(<Catalog …/>, $('#app'));
ReactDOM.render(<Router …><Route component={<Catalog …/>} /></Router>, $('#app'));
```

I don't see a real benefit in exposing the Catalog routing mechanism and I think it adds much more complexity.